gma/src/ios/gma_ios.mm

```diff
@@ -325,43 +323,51 @@ void CompleteAdResult(FutureCallbackData<AdResult>* callback_data,
       ad_error_internal.ad_error_type =
         AdErrorInternal::kAdErrorInternalAdError;
     }
-  } else if (ad_error_internal.code != kAdErrorCodeNone) {
+  } else {
+    FIREBASE_ASSERT_MESSAGE(ad_error_internal.code != kAdErrorCodeNone,
```

Comment:
When would this assert ever be triggered? if error_code is none and error = nil?

Reply:
Yep.  Basically just to ensure that those who call this function fill in the correct parameters in the future, and know immediately if they're misusing it.